### ISSUE TITLE
change links to coursePages

### ISSUE THREAD

**Issue by elbotho** [serlo/db-migrations#381]
In preparation for the coursePage migration we want to change all the links to coursePages to use the new format.
If there are only a handful we could also do that manually.

Since the new format is "backward compatible" (new link format works in current production) this migration could run in production before the actual migration.

Here is the code used to do almost the same thing for grouped exercises:
https://github.com/serlo/db-migrations/blob/db2621f50a50487f3a8320a9588a44bae73d0ec2/src/20240313170000-replace-all-links-to-grouped-exercises.ts

links to the course or the first coursePage should not be changed ( old format: `/{subject}/{course-id}/{slug-from-course-title}`)
All other coursePages should use the new format: `/{subject}/{course-id}/{course-page-id}/{slug-from-course-page-title}`

**Comment by AndreasHuber** [comment]
Is the first coursePage the one with the oldest date @elbotho or do I determine this with some other way?
(https://github.com/serlo/api.serlo.org/pull/1621 currently changes the links to all coursePages)

**Comment by elbotho** [comment]
@AndreasHuber the api does this: `getSortedList(baseUuid.entityChildrenIds)` ([getSortedList](https://github.com/serlo/api.serlo.org/blob/staging/packages/server/src/schema/uuid/abstract-uuid/resolvers.ts#L544-L550)) and then filters trashed pages and pages without an accepted revision ([source](https://github.com/serlo/api.serlo.org/blob/staging/packages/server/src/schema/uuid/course/resolvers.ts#L20-L38))

**Comment by AndreasHuber** [comment]
@elbotho so it sorts by ID instead of date? I imagine the outcome is the same but I can change that.

**Comment by elbotho** [comment]
yeah outcome should be the same. The filtering would be relevant though.
